flocker/node/agents/cinder.py

```diff
@@ -205,52 +325,52 @@ def _blockdevicevolume_from_cinder_volume(cinder_volume):
     :returns: A ``BlockDeviceVolume`` based on values found in the supplied
         cinder Volume.
     """
+    if cinder_volume.attachments:
+        # There should only be one.
```

Comment:
Explain why. I guess openstack only allows a volume to be attached to a single host. Given the interface, this doesn't seem like an essential limitation, so we should handle that case (possibly in a follow-up issue).